resources/services/teams/teams.go

```diff
@@ -120,7 +121,7 @@ func Teams() *schema.Table {
 					{
 						Name:        "team_id",
 						Description: "The id of the name",
-						Type:        schema.TypeBigInt,
+						Type:        schema.TypeString,
```

Comment:
@roneli I don't know whether this should be `TypeBigInt` or `TypeString` -- the generator generates String, so I went with that for now.